### ISSUE TITLE
Spline - Cannot add additional nodes

### ISSUE THREAD

**Issue by AMZN-daimini** [o3de/o3de#12804]
**Describe the bug**
When editing a Spline component, I would expect it to be possible to add additional nodes but I can't seem to find a way to.

**Steps to reproduce**
1. Create an entity
2. Add a Spline Component
3. Edit the Spline

**Expected behavior**
There is a clear way to add new nodes, either from the Component Card or the Component Mode UI.

**Actual behavior**
I cannot add nodes at Edit time.
I would also expect to have better control of the position of nodes, but while I can use the manipulator to move them, I cannot see or set the position via coordinates.

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/82231674/197886125-4a60f519-8ee2-4364-839d-3fa5a94eabf4.png)

**Found in Branch**
Latest development

**Additional context**
Note that it seems to be possible to add new nodes at runtime (the API is there but I haven't tried it)
![image](https://user-images.githubusercontent.com/82231674/197885938-78e24079-18c8-4602-bf04-5bf66ea910e7.png)

**Comment by monroegm** [comment]
@o3de/sig-ui-ux We need a UX design for the functionality that is being requested

**Comment by rainbj** [comment]
@monroegm I have also tried this in the Sig-Ui-UX meeting and wasn't able to make spline. The options to add nodes are not there, or at least in a spot I cannot find. @hultonha - @amzn-leenguy is going to call a meeting about this ticket so we can talk about how best to fix this issue.

**Comment by hultonha** [comment]
https://user-images.githubusercontent.com/82228511/202713498-3f0d69bc-43e9-4842-a3ec-254580e8ac60.mp4

Please see the attached video to show how you can insert vertices

**Comment by hultonha** [comment]
Annoyingly this is not covered in the new O3DE docs (https://www.o3de.org/docs/user-guide/components/reference/shape/spline/) 😖 There was extensive treatment in the original Lumberyard docs when the feature was first added (see here - https://github.com/awsdocs/amazon-lumberyard-user-guide/blob/master/doc_source/component-spline.md#working-with-spline-components).

Tagging @sptramer and @FiniteStateGit for visibility as it would be great to update the docs around this soon.

**Comment by FiniteStateGit** [comment]
I filed https://github.com/o3de/o3de.org/issues/2101 to address this issue.

**Comment by hultonha** [comment]
> I filed [o3de/o3de.org#2101](https://github.com/o3de/o3de.org/issues/2101) to address this issue.

That's great thank you @FiniteStateGit

**Comment by hultonha** [comment]
I think with the doc update issue now opened and the extra info in the ticket this can be closed. If you'd like the updated functionality then opening a separate feature request issue would likely be best. Thoughts @AMZN-daimini?

**Comment by AMZN-daimini** [comment]
Agreed - I think we need a more discoverable UI, but since the functionality is technically there this issue can be resolved with the docs task and we should create a feature request with next steps. Thanks everyone!

**Comment by rainbj** [comment]
@hultonha have you considered using the cluster control and adding a button. This button would allow you to turn on "add point" mode. This is where the cursor could go into vertex add mode. This way, knowing to hold "CTRL" plus click is not as critical. You could also add this shortcut on this button as a tooltip. So shortcut discovery is easier.

**Comment by hultonha** [comment]
That's definitely a possibility @rainbj but we don't have capacity to look at this right now. I suggest opening a new GitHub issue about making inserting new vertices clearer and tag it as `good-first-issue` as it could be something someone from the community or another member of the foundation picks up. Feel free to tag me and I'd be happy to offer support/advice.
